config/import-control.xml

```diff
@@ -7,6 +7,8 @@
     <allow pkg="java.io"/>
     <allow pkg="java.util"/>
     <allow pkg="com.github.checkstyle.regression.data"/>
+    <allow pkg="org.immutables"/>
```

Comment:
Change this to `org.immutables.value`.

`org.immutables.gson` should only be specifically allowed for the class `ModuleExtractInfo` as this conflicts with https://github.com/checkstyle/regression-tool/issues/37#issuecomment-310950569 since gson is related to extract file.

> This component (extract component) ... should contain only code relating to everything needed to  ... read the Checkstyle extract file 

It would be even better if we could do away with this import and signify it as a `TypeAdapters` another way.